lib/graphql-docs/layouts/includes/sidebar.html

```diff
@@ -1,3 +1,7 @@
+<div id="search">
+  <img src="<%= base_url %>/assets/images/search.svg">
+  <input autocomplete="off" placeholder="Search" />
+</div>
```

Comment:
One thing to note about the search behavior - if you filter the list, and then click on something that is in the sidebar, when the page renders, the search input is empty and the full sidebar is shown again. 

I think this behavior is ok, and to fix it would require adding the current search value either to the URL or in the browser's local state.

Please let me know if you think this should be fixed so that when navigating between pages with a search applied, the search value is retained.